src/components/gallery.tsx

```diff
@@ -65,9 +65,15 @@ export default class Gallery extends React.Component<Props> {
     let { items, onKeyDown } = this.props
 
     return (
-      <TransitionGroup className="ars-gallery" onKeyDown={onKeyDown}>
-        {items.map(this.getItem, this)}
-      </TransitionGroup>
+      <>
```

Comment:
What's the fragment doing? It looks like there's already one root element with `<TransitionGroup>`

Reply:
Ah, sorry. Good catch. In an initial attempt, I moved the empty state in here, and wrapped the whole thing in a fragment. This should go away.